packages/realm/src/bson.ts

```diff
@@ -27,6 +27,8 @@ import * as bson from "bson";
 export namespace BSON {
   export const ObjectId = bson.ObjectId;
   export type ObjectId = bson.ObjectId;
+  export const ObjectID = bson.ObjectID;
```

Comment:
Do we want to do this? Some tests did use the `bson.ObjectID` alias

Reply:
I wish it didn't exist, but we might not have a choice.

Reply:
We used to export all of `bson` here, but we agreed [in the technical design](https://docs.google.com/document/d/1mju1TRQAiA0CxXHMs8MYRnvUhqUdkCrh6g-Rb3OUEHk/edit#bookmark=id.krw7alyf7g6j) that we wanted to break this.

```suggestion
```

Reply:
Is there a place I could document this (minor) breaking change? I am mentioning https://github.com/realm/realm-js/issues/4934 just for future reference.